proc-macros/tests/ui/correct/param_kind.rs

```diff
@@ -0,0 +1,63 @@
+use jsonrpsee::{
+	proc_macros::rpc,
+	types::{async_trait, RpcResult},
+	ws_client::*,
+	ws_server::WsServerBuilder,
+};
+
+use std::net::SocketAddr;
+
+#[rpc(client, server, namespace = "foo")]
+pub trait Rpc {
+	#[method(name="method_with_array_param", param_kind="array")]
```

Comment:
Formatting checker won't pick these up since it's macro attributes, but we should be consistent with spacing, plus can drop quotes from `param_kind` now:

```suggestion
	#[method(name = "method_with_array_param", param_kind = array)]
```